### DISCUSSION TITLE
Decentralised forum

### DISCUSSION THREAD

**Discussion by jkutrzeba1** [near/near-sdk-js#356] [Show and tell]
https://github.com/jkutrzeba1/forum-crypto-engine

Running online on:
http://forum-crypto.top/

**Comment by Rezaheydariii** [comment]
hi 
I am getting a "decentralize" error. I feel like I have evaluated all the conditions, but the problem still persists.
 I'm using the latest version of near-sdk-js.
 I have tried to start with a fresh project and reproduce the defect with minimal code changes.
 I have read the console error messages carefully (if applicable).
Description
I have successfully deployed contract on the testnet near.
with comm: near call rezo.testnet checkAndLockBalance --accountId rezo.testnet --amount 1
I am checking and logging the account with this command. However, the transaction is not successful.

:~/near# near call rezo.testnet checkAndLockBalance --accountId rezo.testnet --amount 1
Scheduling a call: rezo.testnet.checkAndLockBalance() with attached 1 NEAR
Receipts: GYi41ibCkkqM3hd3Z5P6ysK2Xcu89WPrW3vsQpW5rdqh, BmQ688NZn21QCoBFYG5fgrzgQxR2wQKKVAYTyyVZpPKE
        Failure [rezo.testnet]: Error: {"index":0,"kind":{"index":0,"kind":{"FunctionCallError":{"CompilationError":{"PrepareError":"Deserialization"}}}}}
near call <contractName> <methodName> [args]

Call method in smart contract

Options:
      --help                            Show help  [boolean]
      --version                         Show version number  [boolean]
  -v, --verbose                         Prints out verbose output  [boolean] [default: false]
      --useAccount, --accountId         Account that will execute the actions  [string] [required]
      --signWithLedger, --useLedgerKey  Use Ledger for signing  [boolean] [default: false]
      --ledgerPath                      HD key path  [string] [default: "44'/397'/0'/0'/1'"]
      --networkId                       Which network to use. Supports: mainnet, testnet, custom  [string] [default: "testnet"]
      --gas                             Max amount of gas this call can use (in gas units)  [string] [default: 30000000000000]
      --deposit, --amount               Number of tokens to attach (in NEAR) to a function call  [string] [default: "0"]
      --depositYocto                    Number of tokens to attach (in yocto NEAR) to a function call  [string] [default: null]
      --base64                          Treat arguments as base64-encoded BLOB.  [boolean] [default: false]
      --privateKey                      AccessKey to use in the call  [string] [default: null]

ServerTransactionError: {"index":0,"kind":{"index":0,"kind":{"FunctionCallError":{"CompilationError":{"PrepareError":"Deserialization"}}}}}
    at parseResultError (/usr/lib/node_modules/near-cli/node_modules/@near-js/utils/lib/errors/rpc_errors.js:31:29)
    at Account.<anonymous> (/usr/lib/node_modules/near-cli/node_modules/@near-js/accounts/lib/account.js:123:56)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/lib/node_modules/near-cli/node_modules/@near-js/accounts/lib/account.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'ActionError',
  context: undefined,
  index: 0,
  kind: { index: 0, kind: { FunctionCallError: [Object] } },
  transaction_outcome: {
    block_hash: '3gJbfBxsecWgs84qwjD3TGrf2auqYuk9TbVYUEJktxu5',
    id: 'EBXPaLQTjK5fpCLgEn633FwQWUECzKxxB5u3Ep5nwo8a',
    outcome: {
      executor_id: 'rezo.testnet',
      gas_burnt: 2427967954614,
      logs: [],
      metadata: [Object],
      receipt_ids: [Array],
      status: [Object],
      tokens_burnt: '242796795461400000000'
    },
    proof: [
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object]
    ]
  }
}
